src/main/resources/db/changelog/changes/02-create-user-table.yml

```
@@ -0,0 +1,41 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-user-table
+      author: Olya Starykova
+      changes:
+        - createTable:
+            tableName: user
```

Table name should be plural
```suggestion
            tableName: users
```